src/main/markdown/eco/add-lib.md

```diff
@@ -0,0 +1,21 @@
+# Update Information
+
+We will add or update information on request. 
+
+## Request for adding Dependencies to the Eco System site
+
+The GWT eco system is a living system. New dependency will be created, other will be end of life. Also we know, that there are
```

Comment:
This paragraph talks about "dependencies", but maybe "libraries" would be more suitable?

Also a small typo "much more" => "many more".